Using Object Storage From 3rd Party Tools.md

```diff
@@ -7,7 +7,7 @@
   "sticky": true
 }}}
 #### Description
-A variety of freeware and shareware FTP clients are available for FTP processes. Examples include Cyberduck, Cloudberry Browser, S3cmd, and others. Additionally, most enterprise backup software has the ability to write directly to an S3 target. For this example, we will show configuration of Cyberduck for bucket configuration. 
+A variety of freeware and shareware FTP clients are available for FTP processes. Examples include Cyberduck, Cloudberry Browser, S3cmd, aupdatednd others. Additionally, most enterprise backup software has the ability to write directly to an S3 target. For this example, we will show configuration of Cyberduck for bucket configuration. 
```

Comment:
Is this a spelling mistake?

Reply:
yes, thanks for catching that, might have been one of those paste-actions where I didn't  notice what I had done.

Reply:
always great to have another pair of eyes looking at this stuff. thanks, again.